easybuild/easyconfigs/p/PyMC3/PyMC3-4.0.0b2-foss-2021b.eb

```diff
@@ -0,0 +1,76 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonBundle'
+
+name = 'PyMC3'
+version = '4.0.0b2'
+
+homepage = 'https://docs.pymc.io/'
+description = """Probabilistic Programming in Python: 
+Bayesian Modeling and Probabilistic Machine Learning with Theano"""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),  # for numpy
+    ('matplotlib', '3.4.3'),
+    #('h5py', '3.6.0'),
+    #('tqdm', '4.62.3'),
+    #('netcdf4-python', '1.5.7'),
+    #('xarray', '0.20.1'),
+    ('typing-extensions', '3.10.0.2'),
```

Comment:
`typing-extensions` is already provided by the Python easyconfig in this toolchain